clang/test/CodeGen/ms-mixed-ptr-sizes.c

```diff
@@ -1,5 +1,6 @@
 // RUN: %clang_cc1 -triple x86_64-windows-msvc -fms-extensions -emit-llvm -O2 < %s | FileCheck %s --check-prefixes=X64,ALL
 // RUN: %clang_cc1 -triple i386-pc-win32 -fms-extensions -emit-llvm -O2 < %s | FileCheck %s --check-prefixes=X86,ALL
+// RUN: %clang_cc1 -triple aarch64-windows-msvc -fms-extensions -emit-llvm -O2 < %s | FileCheck %s --check-prefixes=AARCH64,ALL
```

Comment:
Do we also need some tests in the backend?  I guess most things likely work because we already did most of the work for the aarch64_32 ABI, but I'd like to see some tests that the basics work (load/store/arguments/return values).

Reply:
Backend hasn't been implemented yet - I'll have a follow up PR with the changes and tests.